package.json

```
@@ -5,18 +5,20 @@
   "description": "Личный проект «Шесть городов» от HTML Academy",
   "scripts": {
     "start": "webpack-dev-server",
-    "eslint": "eslint src/**/*.js src/**/*.jsx",
+    "eslint": "eslint src/**/*.js src/**/*.jsx src/**/*.test.js",
```

```suggestion
    "eslint": "eslint src/**/*.js src/**/*.jsx",
```
В принципе под регулярку `eslint src/**/*.js` должны попадать и тесты тоже

---

Тоже так думал, но иначе он их игнорит)